src/common/error/internalErrorCode.ts

```diff
@@ -11,6 +11,7 @@ export enum InternalErrorCode {
         FailedToStartPackager = 106,
         FailedToStopPackager = 107,
         PackagerRunningInDifferentPort = 108,
+        FailedToRestartPackager = 109,
```

Comment:
You will also need to add the string for this error in https://github.com/Microsoft/vscode-react-native/blob/master/errorStrings/errorStrings.json